book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rental
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @person = []
+  end
```

Comment:
- [ ] @rentals should be initialized instead of @person as it represents the association between Book and Rental
```rb
@rentals = []
```